contrib/build-venv.sh

```diff
@@ -9,7 +9,7 @@ EXTRA_ARGS="-Dsystemd=disabled -Dlaunchd=disabled"
 if [ -d /opt/homebrew/opt/libarchive/lib/pkgconfig ]; then
         EXTRA_ARGS="${EXTRA_ARGS} -Dpkg_config_path=/opt/homebrew/opt/libarchive/lib/pkgconfig"
 fi
-if [ ! -d ${BUILD} ]; then
+if [ ! -d ${BUILD} ] || ! [ -e ${BUILD}/build.ninja ]; then
```

Comment:
The first part of the check could also be removed, I guess (`[ -e ${BUILD}/build.ninja ]` will also be silently false if the build dir doesn’t exist). I don’t have an opinion on which is better, happy to change it either way.